src/components/Dashboard/ReservedTokens.tsx

```diff
@@ -143,6 +137,14 @@ export default function ReservedTokens({
           />
         </div>
       )}
+
+      {metadata?.reservedRate ? (
```

Comment:
This is moving the reserved tokens list _below_ the `Distribute tokens` button to be consistent with the `Funding distribution` list.